### DISCUSSION TITLE
Createing the model directory with command

### DISCUSSION THREAD

**Discussion by idanieldrew** [laravel/framework#43221] [Ideas]
- **Laravel Version: 9**

**Description:**
When running `php artisan make:model Post` command,a directory is created in a **app\Models** ,but if a **Model** directory does not exist or has been deleted by mistake,the model created using the previous command is saved in a **app** folder and its namespace is also changed.

 **Proposal:**

Instead of this code in **Illuminate\Foundation\Console\ModelMakeCommand.php**

```
protected function getDefaultNamespace($rootNamespace)
{
     return is_dir(app_path('Models')) ? $rootNamespace.'\\Models' : $rootNamespace;
}
```
change to this state

```
protected function getDefaultNamespace($rootNamespace)
{
     return $rootNamespace.'\\Models';
}
```

**Comment by henzeb** [comment]
As I understand it, this is something Otwell wasn't really happy with. The current situation is a compromis.

You could however propose a configuration option instead, where the default is no folder.
